api/api_ethereum.go

```diff
@@ -1200,6 +1200,12 @@ func (args *EthTransactionArgs) ToMessage(globalGasCap uint64, baseFee *big.Int,
 	// Reject invalid combinations of pre- and post-1559 fee styles
 	if args.GasPrice != nil && (args.MaxFeePerGas != nil || args.MaxPriorityFeePerGas != nil) {
 		return nil, errors.New("both gasPrice and (maxFeePerGas or maxPriorityFeePerGas) specified")
+	} else if args.GasPrice == nil && args.MaxFeePerGas == nil {
```

Comment:
Is this an error condition?
I think if those two values are undefined by user, we can set value in below logic.

Reply:
this is toMessage function. Please ignore this!